SU-SWS/stanford_profile_helper
# D8CORE-2201, D8CORE-2448: hiding the default external link

# READY FOR REVIEW


# Summary
When we add the extlink module, it applies the external `<svg>` to _all_ external links. This PR visually hides it so we can use our own styles. This keeps the screen reader text "link is external" thus helping to keep our links accessible. 

This also styles the External Link's existing mailto icon.

# Needed By (Date)
- Soon

# Urgency
- not urgent

# Steps to Test

1. Check out this branch,
1. Clear the cache
1. Verify that the external link <svg> does not show on all links, but the aria-label "link is external" is applied to every link. 
1. Verify that the external link style shows up in WYSIWYG text fields.

![Screen Shot 2020-08-06 at 5 51 27 PM](https://user-images.githubusercontent.com/284440/89597291-8f16cf80-d80e-11ea-8a14-a6bc772ac305.png)


# Affected Projects or Products
- D8Core

# Associated Issues and/or People
- JIRA ticket https://stanfordits.atlassian.net/browse/D8CORE-2448
https://stanfordits.atlassian.net/browse/D8CORE-2201
https://stanfordits.atlassian.net/browse/D8CORE-2492
- Other PRs: https://github.com/SU-SWS/stanford_profile/pull/261
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
